src/rendering/renderers/shared/texture/RenderableGCSystem.ts

```diff
@@ -187,4 +195,14 @@ export class RenderableGCSystem implements System<RenderableGCSystemOptions>
         this._renderer = null as any as Renderer;
         this._managedRenderables.length = 0;
     }
+
+    private _removeRenderable(renderable: Container): void
```

Comment:
I might be missing something here, but since `addRenderable()` takes a `Renderable`, why is `_removeRenderable()` taking a `Container`?

Reply:
all events expect the thing passed to be a `Container`. We don't have any fancy types that know what the thing you listening too is

Reply:
I suppose that makes sense with the assumption that anything passed into this extends `Container`, though I'm a bit nervous that the add only takes a Renderable. If there's never the possibility of a Renderable that doesn't extend Container then this is probably fine, though seems sketchy type-wise.